worker/package.json

```diff
@@ -47,6 +47,7 @@
     "aws4fetch": "^1.0.18",
     "dateformat": "^5.0.3",
     "events": "^3.3.0",
-    "itty-router": "^4.0.13"
+    "itty-router": "^4.0.13",
+    "zod": "^3.23.8"
```

Comment:
Can zod be a dev dependency? We want to keep the worker size as small as possible.